web/src/layout/Header/DesktopHeader.tsx

```diff
@@ -90,6 +90,19 @@ const DesktopHeader = () => {
   const [isDappListOpen, toggleIsDappListOpen] = useToggle(false);
   const [isHelpOpen, toggleIsHelpOpen] = useToggle(false);
   const [isSettingsOpen, toggleIsSettingsOpen] = useToggle(false);
+  const [initialTab, setInitialTab] = useState<number>(0);
+  const location = useLocation();
+
+  const initializeNotifications = () => {
```

Comment:
better if we use `useCallback` here to avoid re-defining the function each render